src/toast/pipeline_tools/pointing.py

```diff
@@ -0,0 +1,156 @@
+# Copyright (c) 2019 by the parties listed in the AUTHORS file.
+# All rights reserved.  Use of this source code is governed by
+# a BSD-style license that can be found in the LICENSE file.
+
+import argparse
+import os
+
+import numpy as np
+
+from ..timing import function_timer, Timer
+from ..utils import Logger, Environment
+
+from ..map import OpMadam, OpLocalPixels, DistPixels
+
+from ..tod import OpPointingHpix
+
+
+def add_pointing_args(parser):
+    """ Add the pointing arguments
+    """
+
+    # `nside` may already be added
+    try:
+        parser.add_argument(
+            "--nside", required=False, default=512, type=np.int, help="Healpix NSIDE"
+        )
+    except argparse.ArgumentError:
+        pass
+    # `coord` may already be added
+    try:
+        parser.add_argument(
+            "--coord", required=False, default="C", help="Sky coordinate system [C,E,G]"
+        )
+    except argparse.ArgumentError:
+        pass
+    parser.add_argument(
+        "--hwp-rpm",
+        required=False,
+        default=0,
+        type=np.float,
+        help="The rate (in RPM) of the HWP rotation",
+    )
+    parser.add_argument(
+        "--hwp-step",
+        required=False,
+        default=None,
+        help="For stepped HWP, the angle in degrees of each step",
```

Comment:
what about adding unit to argument name? i.e. `--hwp-step-deg`?

Reply:
I like it. Also `--hwp-step-time` --> `--hwp-step-time-s`